packages/components/src/internal/app/actions.ts

```diff
@@ -73,13 +75,18 @@ export function setReloadRequired() {
 export function menuInit(currentProductId: string, userMenuProductId: string, productIds?: List<string>) {
     return (dispatch, getState) => {
         let menu = getState().routing.menu;
-        if (!menu.isLoaded && !menu.isLoading) {
-            dispatch({
-                type: MENU_LOADING_START,
-                currentProductId,
-                userMenuProductId,
-                productIds, // when undefined, this returns all menu sections for modules in this container
-            });
+        if ((!menu.isLoaded && !menu.isLoading) || menu.needsReload) {
+            if (menu.needsReload) {
+                resetQueryGridState(); // when import is completed, force reset for all grids
+            }
+            else if (menu.needsReload) {
```

Comment:
I think the `if (menu.needsReload)`  needs to be removed here.

Reply:
good catch, this broke the whole menu. fixed.